### ISSUE TITLE
Versioning on CurseForge not correct 

### ISSUE THREAD

**Issue by Ratomas** [Jaliborc/Scrap#189]
Versioning on CurseForge not correct 9.2.7 is showing as the latest when 10.0.2 is actually the latest. Every day I have to reinstall the 10.0.2 release version because 9.2.7 keep being installed as the latest release version.

**Comment by schnoodly** [comment]
Issue as described in #165 but for some reason marked as solved. To be clear, this is for BAGNON SCRAP, not normal Scrap.

**Comment by Ratomas** [comment]
Yes but Bagnon_Scrap directs us here for issues

**Comment by schnoodly** [comment]
Indeed. It's fairly confusing, as there obviously is some sort of difference - case and point, the aforementioned issue's confusion on the matter. 

I just ended up uninstalling it entirely, got annoying

**Comment by Taurmin** [comment]
I only discovered today that there is actually an updated version of Bagnon Scrap, I spent a good couple of days trying to find a new an alternative bag solution solely because the Bagnon integration seemed to have been abandonned.
